contracts/vesting/src/errors.rs

```diff
@@ -4,7 +4,7 @@ use thiserror::Error;
 
 #[derive(Error, Debug, PartialEq)]
 pub enum ContractError {
-    #[error("{0}")]
+    #[error("{} - {0}", line!())]
```

Comment:
This looks like something we could use on more than one error type, if it's really needed

Reply:
I was chasing down an error, that was not even coming from the contract, pretty sure we don't really need it